### DISCUSSION TITLE
Fetch some data whenever page reloads?

### DISCUSSION THREAD

**Discussion by Jasenkoo** [vercel/next.js#11502] [Help]
Hello everyone, I am new to the whole React world and Next.js however I have a lot of experience working with Vue.js and Nuxt.js (vue's alternative to next). There we have a function called nuxtServerInit which will only trigger when page is reloaded, or on initial application load.

This means that when I am on some page and reload that function gets called and then the page specifics ssr method. I was wondering if there is something similar I can achieve here?

**Comment by timneutkens** [comment]
The equivalent is probably https://nextjs.org/docs/basic-features/data-fetching#getserversideprops-server-side-rendering
